phoenix-core/src/it/java/org/apache/phoenix/end2end/PropertiesInSyncIT.java

```diff
@@ -402,16 +486,17 @@ private String createBaseTableWithProps(Connection conn) throws SQLException {
      * @param baseTableName Name of the HBase base table on which to create an index
      * @param indexType LOCAL, GLOBAL or if we pass in null as the indexType,
      *                 we create a view and an index on that view for the given base table
+     * @param returnLocalIndex if true, when creating a local index, return the local index name
+     *                         of the base table name instead of the physical base table name
      * @return The physical HBase table corresponding to the index created
```

Comment:
This doesn't always return the physical HBase table that stores the index being created right? Might be a cleaner if you return a pair of the index name and the index physical table name.